pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -129,6 +129,12 @@ in
     ];
   };
 
+  pg_query = attrs: {
+    postPatch = ''
+      ln -s ${libpg_query}/lib ${libpg_query.name}
```

Comment:
Oh, OK! I’ll look at this over the weekend.

What would we do, though, if we had two projects with each depending on a _different_ version of `pg_query`? We could depend only on _one_ version of `libpg_query` in this override, right? What then?